doc/source/flexibility/maximum_binlog_size.rst

```diff
@@ -0,0 +1,57 @@
+.. _maximum_binlog_size:
+
+========================================
+ Restricting the number of binlog files
+========================================
+
+Estimating the number of binary logs and disk space use is a challenge. |Percona Server| has the variable :variable:`binlog_space_limit`, which sets the maximum amount of space to use for all binary logs. With the :variable:`binlog_space_limit` set to `0`, the default value, the variable is turned off and no limit is set. You set the :variable:`binlog_space_limit` on and provide a size limit. The value specified is not a hard limit. A single transaction is written to a single binary log, and the size of this transaction may exceed the limit.
+
+For example, the :variable:`max_binlog_size` is set to 1GB, which is the default value and the maximum value and the :variable:`binlog_space_limit` is set to 10GB. A large transaction changes 5GB of data. To store the transaction, the :variable:`binlog_space_limit` removes old binary logs as needed, up to the three oldest ones, to remain under the limit.
+
+Example
+=======
+
+.. code-block:: guess
+
+[mysqld]
+max_binlog_size = 1G
+binlog_space_limit = 10G
+
+Version Specific Information
+============================
+
+  * 5.7.10-1:
+    Variable :variable:`max_binlog_files` ported from |Percona Server| 5.6.
```

Comment:
Why are there 5.7 versions mentioned in an 8.0 documentation? Especially about a removed variable.

Reply:
removed.